ngraph/test/onnx/onnx_import.in.cpp

```diff
@@ -2713,6 +2713,32 @@ NGRAPH_TEST(${BACKEND_NAME}, onnx_model_scatterND_const_i32_indices)
     test_case.run();
 }
 
+NGRAPH_TEST(${BACKEND_NAME}, onnx_model_gather_elements_float)
+{
+    const auto function = onnx_import::import_onnx_model(
+        file_util::path_join(SERIALIZED_ZOO, "onnx/gather_elements_float.prototxt"));
+    auto test_case = test::TestCase<TestEngine>(function);
+
+    test_case.add_input<float>({1, 7, 4, 3});
+    test_case.add_input<int64_t>({1, 1, 0, 1, 0, 1});
+    test_case.add_expected_output<float>(Shape{2, 3}, {7, 7, 1, 3, 4, 3});
+
+    test_case.run();
+}
+
+NGRAPH_TEST(${BACKEND_NAME}, onnx_model_gather_elements_int32)
+{
+    const auto function = onnx_import::import_onnx_model(
+        file_util::path_join(SERIALIZED_ZOO, "onnx/gather_elements_int32.prototxt"));
+    auto test_case = test::TestCase<TestEngine>(function);
+
+    test_case.add_input<int32_t>({1, 2, 3, 4, 5, 6, 7, 8, 9});
+    test_case.add_input<int64_t>({1, 0, 1, 1, 2, 0});
+    test_case.add_expected_output<int32_t>(Shape{2, 3}, {4, 2, 6, 4, 8, 3});
+
+    test_case.run();
+}
+
```

Comment:
I think we need more tests for this op. The ones below the op definition https://github.com/onnx/onnx/blob/master/docs/Operators.md#GatherElements could be used as test cases (in particular I mean gathering over a non-default axis and a test or two with negative indices). Here are some tests for this op too https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/test/providers/cpu/tensor/gather_elements_op_test.cc

Reply:
Non-default axis case is already covered with [onnx_model_gather_elements_float](https://github.com/openvinotoolkit/openvino/pull/3822/files#diff-6636d199b3570fa13f64406116bbe01c1876f6c285ecb3d89ea9f1212d988360R11) test.
As of adding more test cases - the examples from ONNX spec are similar to committed ones, while the tests from ONNXRuntime exceed the scope of ONNX Importer scope imo - let's discuss it.